UDP/course_materialized_view.md

```diff
@@ -22,6 +22,9 @@ on pe.person_id = cse.person_id join entity.person p on p.person_id = cse.person
 courses_enrollment as (select cst.*, e.person_id, e.uniqname, e.person_name from courses_sections_of_current_term cst join enrollment e on e.course_section_id = cst.course_section_id),
 course_grades as (select ce.*, cg.le_current_score as "Current_Grade", cg.le_final_score as "Final_Grade", cg.gpa_cumulative_excluding_course_grade as cumulative_gpa from entity.course_grade cg join courses_enrollment ce on ce.course_section_id = cg.course_section_id and ce.person_id = cg.person_id),
```

Comment:
If you look here were are only pulling `current_score` grade that are visible to student as per [documentation](https://docs.udp.unizin.org/ucdm/data-dictionary-2.0.html#course_grade)

Reply:
Yes! I checked that, too!